### PR TITLE
Added in a few tweaks

### PR DESCRIPTION
**Pull request by apeltzer**
See the PR for details :) 

@netlify /docs/guidelines/regulatory/checklist

### CODE REVIEW

File: sites/docs/src/content/docs/guidelines/regulatory/overview.md [hunk 1]
```diff
@@ -37,20 +37,22 @@ Computer system validation (CSV) is the broad term that encompasses that your so
...
-These entities, such as the U.S. Food and Drug Administration (FDA) consensus standards, the European Medicines Agency (EMA)/ European Commission, and others, are responsible for ensuring the safety, efficacy, and quality of drugs/tests before they reach the market.
-Each of them has its own set of standards that need to be followed, depending on the intended use of the test.
+Regulatory authorities play a pivotal role in the pharmaceutical industry, particularly in the context of drug approval. 
+These entities, such as the U.S. Food and Drug Administration (FDA) consensus standards, the European Medicines Agency (EMA)/ European Commission, and others, are responsible for ensuring the safety, efficacy, and quality of drugs and medical tests before they reach the market.
+Each of them has its own set of standards that need to be followed, depending on the intended use of the test. We differentiate between standards that apply on a software level versus on a more infrastructure & computer systems validation level. The latter (CSV) is out of scope for this guidance and should be dealt with depending on your infrastructure or IT provider.
```
Comment (kenibrewer) [inline sites/docs/src/content/docs/guidelines/regulatory/overview.md:39]:
```suggestion
Each of them has its own set of standards that need to be followed, depending on the intended use of the test. We differentiate between standards that apply on a software level versus on a more infrastructure & computer systems validation level.
The latter (CSV) is out of scope for this guidance and should be dealt with depending on your infrastructure or IT provider.
```

File: sites/docs/src/content/docs/guidelines/regulatory/checklist.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## Security
+
+## Documentation
+
+## Standard operating procedures (SOPs)
+
+## Functional testing
+
+## Integrative testing
+
```
Comment (tomiles) [inline sites/docs/src/content/docs/guidelines/regulatory/checklist.md:37]:
```suggestion
##  Analytical performance testing
```

**Comment by apeltzer** [comment]
@nf-core-bot fix linting

**Review by kenibrewer — APPROVED** [review]

**Comment by apeltzer** [comment]
@nf-core-bot fix linting

**Review by tomiles — APPROVED** [review]

**Pull request merged**